Changelog.md

```diff
@@ -11,6 +11,15 @@ To get all details, and changes at all versions, including development ones, use
 You are reminded that Watai uses [SemVer](http://semver.org), which means upgrades that have only a patch number (last digit) change are backwards-compatible, and versions with a minor number (second digit) are API-breaking while in `0` major versions.
 
 
+v0.7 (WIP)
+----
+
+#### Minor changes
```

Comment:
Well, if I have to rename all my files I consider it not to be that minor ;)
Is there any backward-compatibility for names?